docs/components/modeler/web-modeler/additional-features/call-activity-linking.md

```diff
@@ -0,0 +1,27 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Call activity linking
+---
+
+You can use one of the following approaches to link a [call activity](/components/modeler/bpmn/call-activities/call-activities.md) to another process.
+
+### Using the link button
+
+If you select a call activity, a link button appears at the bottom right. Clicking on the button opens a dialog from which you can choose any diagram from the same project. Click on the `Link` button after selecting the diagram. This will create the link by copying the process ID of the diagram to the `Called element` section in the properties panel.
+
+![overlay](img/overlay.png)
+
+For call activities that are already linked, the button appears even when the element is not selected. Clicking on the button opens a dialog which shows the diagram that the call activity is linked to. It is possible to navigate to the linked diagram by clicking on it or you can use the `Unlink` button to remove the link.
+
+![overlay](img/linked.png)
+
+### Using the properties panel
+
+You may also enter the process ID directly in the `Called element` section in the properties panel.
```

Comment:
Add some reasoning: "... in the properties panel, e.g. when you call a process from another project."

Reply:
I'm not sure if we should add such a reasoning. This is just another way of linking and can be used also to link a process from the samem project.